arches/app/models/models.py

```diff
@@ -668,7 +668,9 @@ def get_functions_x_graphs(self, force_recalculation=False):
                 for function_dict in function_slugs
             ]
         else:
-            return self.functionxgraph_set.all()
+            return [
```

Comment:
why this change?

Reply:
I sent you a message about this on slack; it's because of the call to append to the output of this function.  It'll blow up if that's a queryset.